addons/mail/static/src/components/discuss/tests/discuss_moderation_tests.js

```diff
@@ -1227,6 +1227,130 @@ QUnit.test('new pending moderation message posted by someone else when moderator
     );
 });
 
+QUnit.test('accept multiple moderation messages', async function (assert) {
+    assert.expect(9);
+
+    Object.assign(this.data.initMessaging, {
+        channel_slots: {
+            channel_channel: [{
+                id: 20,
+                is_pinned: true,
+                channel_type: "channel",
+                name: "general",
+                moderation: true,
```

Comment:
alphabetical order (+ technical string with single quotes `channel_type`, double quotes for UI strings `name`)